share_light.install

```diff
@@ -106,6 +106,35 @@ function _share_light_default_channels() {
   return $channels;
 }
 
+/**
+ * Allow proprietary share url protocols in links.
+ */
+function share_light_update_7201() {
```

Comment:
This must also be called in `hook_install()` otherwise it won’t be executed for new installations.